modules/misc/config.py

```diff
@@ -135,7 +135,7 @@ def from_json(cls, data: JSON) -> Self:
         """Creates a new Config object from the JSON data contained in the user config file."""
 
         return cls(
-            general=data.get("general", dict()),  # type:ignore
+            telemetry_buffer_size=data.get("telemetry_buffer_size", int()),
```

Comment:
I think you could pass just `None` instead of a default option of `int`. This would use the default value of 20 that you set above. Idk what `int()` does in this case but I think sticking with the default is wiser.

Reply:
Corrected to int(20) to pass in default value of 20 if config is read and the key is missing.